pkg/scaling/resolver/scale_resolvers.go

```diff
@@ -40,6 +43,18 @@ const (
 	referenceCloser   = ')'
 )
 
+var kedaNamespace, _ = util.GetClusterObjectNamespace()
+
+// getWatchNamespace returns the namespace the operator should be watching for changes
+func getRestrictSecretAccess() bool {
```

Comment:
WDYT about naming this as `isSecretAccessRestricted()`? It's clearer semantically

Reply:
Good suggestion, updated